extensions/query-store/src/common/constants.ts

```diff
@@ -23,6 +27,7 @@ export function plan(queryId: string): string { return localize('plan', "Plan {0
 export function topResourceConsumingQueriesToolbarLabel(databaseName: string): string { return localize('topResourceConsumingQueriesToolbarLabel', "Top 25 resource consumers for database {0}", databaseName); }
 
 export const configure = localize('configure', "Configure");
+export const openInNewTab = localize('openInNewTab', "Open in new tab");
```

Comment:
Should this be "Open In New Tab" instead?
![image](https://github.com/microsoft/azuredatastudio/assets/57200045/2bba3468-2fbf-4ee8-80e7-7148ac7e2743)

Reply:
updated! 
![image](https://github.com/microsoft/azuredatastudio/assets/31145923/a79b5faf-4c10-4276-83c2-87e29823243e)